.github/workflows/db-ops.yaml

```
@@ -0,0 +1,206 @@
+      - name: Get Latest Commit ID
+        id: lastcommit
+        uses: nmbgeek/github-action-get-latest-commit@main
+        with:
+          owner: ${{ github.repository_owner }}
+          token: ${{ secrets.SUBMODULES_TOKEN }}
+          repo: wf-data-migration
+          branch: ${{ env.MIGRATION_REF }}
+
+      - name: Cache Docker layers
+        id: cache
+        uses: actions/cache@v2
```

actionlint

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)